### ISSUE TITLE
You should be notified when you have been upgraded to a new version of the script

### ISSUE THREAD

**Issue by ghost** [nathan-osman/auchatstatus#6]
Once you have successfully been upgraded to a new version of the script I feel that you should still get some sort of notification like before that offers a link to the changelog and also says that you can start using the new version by refreshing the page.

**Comment by ByteCommander** [comment]
Agree.

**Comment by nathan-osman** [comment]
Yeah, it would be a great idea if the script could notify users that a new version is available and remind them to reload the page.
